### ISSUE TITLE
Add instructions on how to use OpenCastSwift in a new project

### ISSUE THREAD

**Issue by kottkrig** [mhmiles/OpenCastSwift#2]
It would be great if there were a few pointers in the Readme on how to get a project with OpenCastSwift up and running.

Is the preferred way to use Carthage?

**Comment by WontSeeMeComin** [comment]
Agreed, I am lost here. How do we actually integrate this into an Xcode project?

**Comment by coucoseth** [comment]
was an xcode example ever availed or does anyone know how to use this library because the documentation doesnt make sense to me.
